src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("6.1.0.0")]
-[assembly: AssemblyFileVersion("6.1.0.0")]
+[assembly: AssemblyVersion("6.1.1.0")]
```

Comment:
don't we need to increment the version? Should we change version from 6.1.0.0 to 6.0.0.0 in AssemblyInfo,cs and in project.json?